azure-pipelines.yml

```diff
@@ -27,34 +27,6 @@ resources:
       name: xamarin/yaml-templates
       endpoint: xamarin
 
-trigger:
```

Comment:
Could you revert this unnecessary changes?. If need help reverting etc just let me know.